colcon_core/command.py

```diff
@@ -126,7 +124,6 @@ def main(*, command_name='colcon', argv=None):
 
 
 def _main(*, command_name, argv):
-    global colcon_logger
```

Comment:
I think in terms of the logger it's possible to remove the `global` prefix since we never modify the value of it directly but through methods of the same logger.